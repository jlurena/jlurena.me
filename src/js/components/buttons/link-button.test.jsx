/* eslint-env jest */

import React from 'react'
import renderer from 'react-test-renderer'
import { LinkButton } from '.'

describe('<Button/>', () => {
  const component = renderer.create(
    <LinkButton aria-label='Open in new tab' href='https://example.com' target='_blank'>Hello</LinkButton>
  )

  const tree = component.toJSON()

  test('Rendering', () => {
    expect(tree).toMatchSnapshot()
  })
})
