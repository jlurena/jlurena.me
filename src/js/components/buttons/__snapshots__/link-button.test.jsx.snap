// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Button/> Rendering 1`] = `
<div
  className="container"
>
  <a
    aria-label="Open in new tab"
    href="https://example.com"
    target="_blank"
  >
    Hello
  </a>
</div>
`;
